src/test/java/resources/application.properties

```
@@ -0,0 +1,8 @@
+spring.application.name=spring-project
+spring.datasource.url=jdbc:mysql://localhost:3306/mydatabase?serverTimezone=UTC
+spring.datasource.username=root
+spring.datasource.password=pass1234
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.show-sql=true
```

let's check task description
```suggestion
spring.datasource.url=jdbc:h2:mem:testdb
spring.datasource.driverClassName=org.h2.Driver
spring.datasource.username=sa
spring.datasource.password=password
spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
```